### DISCUSSION TITLE
Account @SteveLauC "suspended"

### DISCUSSION THREAD

**Discussion by ghost** [nix-rust/nix#2458] [General]
Hi @asomers, I am steve (@SteveLauC), you will notice that account simply disappeared, for the reason that it has been marked as spammy, and possibly due to this discussion: https://github.com/nix-rust/nix/discussions/2445. (You cannot open that link because it is created by me and thus has been removed.)

I can still access the account (push to the repo, merge PRs), but I cannot open issue/PR and communicate with others through comments, not sure when it will recover...

**Comment by asomers** [comment]
Shoot.  I hope I wasn't responsible.  I closed that discussion as off-topic, but I now see that it wasn't the original poster (@stevelauc) who was off-topic, but the second poster.  Does Github have a procedure to reactivate your account?

**Reply by ghost** [reply]
>  I hope I wasn't responsible.

No need to worry about it.

> Does Github have a procedure to reactivate your account?

I have opened a ticket to reinstate that account.

Given that I can still push to the repo/merge PRs, I think everything is still ok.

**Comment by SteveLauC** [comment]
Ok, I am back, they reinstated my account.

**Comment by asomers** [comment]
Great news! Did you figure out what the original problem was?

**Reply by SteveLauC** [reply]
No, they said they will give it a further investigation:

<img width="982" alt="Screenshot 2024-07-08 at 8 40 10 AM" src="https://github.com/nix-rust/nix/assets/96880612/9d79c1f7-ee88-457e-8de5-d5932fca58a5">
